vllm/model_executor/models/interfaces.py

```diff
@@ -11,11 +12,12 @@
 
 if TYPE_CHECKING:
     from vllm.attention import AttentionMetadata
-    from vllm.multimodal.inputs import NestedTensors
     from vllm.sequence import IntermediateTensors
 
 logger = init_logger(__name__)
 
+T = TypeVar("T")
```

Comment:
If we use `typing_extensions.TypeVar`, then we can set a default for this variable to provide some hints.